walk-in-interview/src/main/java/com/google/job/data/JobsDatabase.java

```diff
@@ -42,6 +43,17 @@
                 .set(job);
     }
 
+    /**
+     * Updates a specific field.
```

Comment:
Maybe update the comment to explain a bit clearer on the intention of this method as there're two job id fields we're referring to.

Reply:
Fixed